experiment/build/gcb_build.py

```diff
@@ -19,11 +19,21 @@
 from common import benchmark_utils
 from common import experiment_path as exp_path
 from common import experiment_utils
+from common import filesystem
+from common import fuzzer_config_utils
 from common import logs
 from common import new_process
 from common import utils
+from common import yaml_utils
 from experiment.build import build_utils
 
+BUILDER_STEP_IDS = [
```

Comment:
I was having trouble wrapping my head around how cloud build worked. It looked like the yaml files were the only way that we configure what's happening there, so did we want to append the arguments to those configs? If so, are these the correct places to append them?

If not let me know, this should be easy enough to remove.

Reply:
I wonder how this works given that we use multiprocessing to build?
Sorry, but I realized yesterday that this model probably requires more dynamism than the config files allow.
I think we probably need to switch to something like this: https://github.com/google/oss-fuzz/blob/dbe2731d93f8de04969543e45338af26a6dc3773/infra/gcb/build_base_images.py

Reply:
Something like the oss-fuzz implementation does look much cleaner but I don't see what missing functionality it provides (just want to make sure I'm not missing something there). Other than that, are these the places we'd want to be plugging the arguments from the config files in? Is anything else missing?

Reply:
1. I think when write a new config file it needs to be in a temp directory, otherwise doing this in parallel breaks right?
2. I don't think it is theoretically more powerful than this approach, just cleaner and easier to get right.

Reply:
1. I think a temp directory makes sense either way. There's no reason for it to be in the configs. That said, I'm not sure that this would break as it should be a unique filename. I might be missing something here.
2. I'm not too confident in my understanding of cloud build to refactor this file to use the other API throughout this file. Would you mind taking the config file rewrite approach for now with a TODO to refactor that in a follow up once the configs are working?

Reply:
1. Sorry, I didn't look at the code closely, enough, I think the approach you took doesn't suffer from parallelism issues (though we may need to ensure that no two variants have the same name, unless we append the name to the beginning can't remember right now.
2. Yes I can accept this for now.

Reply:
1. Switched to a temp filename anyway. From what I can tell this happens after the config directory sync so there's no reason to handle it the older way.
2. Added a TODO.